src/server.rs

```diff
@@ -1,18 +1,25 @@
-use crate::processor::handle_request;
+use crate::processor::{apply_headers, handle_request};
 use crate::router::Router;
-use std::process;
+use crate::types::Headers;
+use std::sync::atomic::AtomicBool;
+use std::sync::atomic::Ordering::{Relaxed, SeqCst};
 use std::sync::Arc;
+use std::thread;
 // pyO3 module
+use actix_web::*;
```

Comment:
Can we switch this to individual imports instead of importing everything?

Reply:
can do, its actix convention in the docs to import all

Reply:
@JackThomson2 , please change it to import specific items instead of an import all and then we can merge this PR.